src/app/src/components/FacilityListItemsTable.jsx

```diff
@@ -73,251 +73,285 @@ const createSelectedStatusChoicesFromParams = (params) => {
     return null;
 };
 
-function FacilityListItemsTable({
-    list,
-    items,
-    filteredCount,
-    fetchingItems,
-    selectedFacilityListItemsRowIndex,
-    makeSelectListItemTableRowFunction,
-    fetchListItems,
-    match: {
-        params: {
-            listID,
-        },
-    },
-    history: {
-        push,
-        location: {
-            search,
-        },
-    },
-}) {
-    const {
-        page,
-        rowsPerPage,
-    } = createPaginationOptionsFromQueryString(search);
 
-    const params = createParamsFromQueryString(search);
+class FacilityListItemsTable extends Component {
+    componentDidUpdate(prevProps) {
+        const {
+            items,
+            fetchListItems,
+            match: {
+                params: {
+                    listID,
+                },
+            },
+            history: {
+                location: {
+                    search,
+                },
+            },
+        } = this.props;
 
-    const handleChangePage = (_, newPage) => {
-        push(makePaginatedFacilityListItemsDetailLinkWithRowCount(
-            listID,
-            (newPage + 1),
-            rowsPerPage,
-            params,
-        ));
-        fetchListItems(listID, newPage + 1, rowsPerPage, params);
-    };
+        const params = createParamsFromQueryString(search);
+        if (params.status && params.status.includes('POTENTIAL_MATCH') && items && prevProps.items) {
```

Comment:
Do we have `POTENTIAL_MATCH` set as a constant somewhere in the app? If so we could consider checking on the constant value here; if not, we could add one?

Reply:
Looks like it's set on the `facilityListItemStatusChoicesEnum`.

Reply:
Addressed in fixup ca65c8c